samples/python/digits_video.py

```diff
@@ -98,3 +99,4 @@ def main():
 
 if __name__ == '__main__':
     main()
+cv2.destroyAllWindows()
```

Comment:
This line should be places into `main()` function.

Looks like this observed effect is from IPython or similar. Normal standalone Python applications works well.

BTW, You could use `destroyWindow(name)` call instead:
https://stackoverflow.com/questions/15144442/when-is-an-opencv-imshow-window-destroyed-c

Reply:
Yes destroyAllWindows() can be called in main() function . Thanks